mathesar_ui/src/stores/table-data/tabularData.ts

```diff
@@ -99,6 +99,9 @@ export class TabularData {
     this.columnsDataStore.on('columnPatched', async () => {
       await this.recordsData.fetch();
     });
+    this.columnsDataStore.on('columnsFetched', async () => {
+      this.display.isTableInspectorVisible.set(true);
+    });
```

Comment:
Why do we need to set the table inspector to be visible after fetching the columns?

Reply:
This is to make the inspector visible as soon as the table view is opened.

Reply:
I opened #1616 to propose deleting these lines. We can continue the discussion there if there's a reason to keep the lines that I'm not seeing.